### DISCUSSION TITLE
At Your Service

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#29] [Card Balance] [answered]
ANNIHILATED 
 Draw a random card from your discard pile and Annihilate this card

Ok, this is out of hand. 1DP cost. All i ask.

**Comment by 21-ric** [comment]
I will take "reassignment" for example.
During your first turn you play "reassignment" then you play "at your service" to draw again "reassignment".

If we add 1 [DP] to "at your service.
During your first turn you play "reassignment" then you play "at your service" for 1[DP] to draw again "reassignment".

If we take @TomekPL-1's idea : annihilate one card in your discard pile then draw a random card in your discard pile.
During your first turn you play "reassignment" then you can't play "at your service" to draw again "reassignment" because you must have 2 cards in your discard pile. If you want to draw "reassignment" you must have 2 cards in your discard pile. 50% to draw this card 50% to annihilate this card.

**Reply by Sol-au-fein** [reply]
Sounds like good idea.

**Reply by Michalx11** [reply]
Allright letd try this idea

**Reply by framerate** [reply]
> Allright letd try this idea

Which one? There are two, right?

**Reply by TomekPL-1** [reply]
Justin, I think that they were talking about this option:

At Your Service (GENERAL) [U]
ANNIHILATED
Play if you have at least two cards in your discard pile Annihilate one random card from discard pile then draw one random card from same pile.

And with this solution it would be ok for me if 'At your service' did not have Annihilated key word (after use)

**Reply by framerate** [reply]
Yeah that sounds pretty doable to me!

**Reply by Sol-au-fein** [reply]
To be fair, who cares about annihilating card in discard pile? People will use it anyway with no remorse. In the end it is still free card from discard pile.

**Comment by Michalx11** [comment]
I like @TomekPL-1 idea but this would limit its use to Vince, Coral and Doctor Diana only. 
Actually if only UNIQUE warriors can use it (not a bad idea) then we dont really need to move it to faction

**Reply by TomekPL-1** [reply]
This is about miscommunication?

**Reply by framerate** [reply]
I think @Michalx11 is posting on the wrong topic, maybe?

**Reply by Sol-au-fein** [reply]
Please remove non-related comments 🤡
@Michalx11

**Comment by Michalx11** [comment]
Haha yee

**Reply by Michalx11** [reply]
I did by accident... I copy/paste it to Mjsscommunications discussion

**Reply by Sol-au-fein** [reply]
🤡

**Comment by framerate** [comment, marked as answer]
Going in as 
> Play if you have at least two cards in your discard pile Annihilate one random card from discard pile then draw one random card from same pile.
